ui/selectors/selectors.js

```diff
@@ -790,15 +790,3 @@ export function getIsMultiLayerFeeNetwork(state) {
 export function getAdvancedGasFeeValues(state) {
   return state.metamask.advancedGasFee;
 }
-
-/**
- *  To check if the user has set advanced gas fee settings as default with a non empty  maxBaseFee and priotityFee.
- *  @param {*} state
- *  @returns Boolean
- */
-export function getIsAdvancedGasFeeDefault(state) {
-  const { advancedGasFee } = state.metamask;
```

Comment:
Removing this method because we are not using it and it is easier to just use `Boolean(getAdvancedGasFeeValues())`